pool/db.go

```diff
@@ -52,6 +52,8 @@ var (
 	csrfSecret = []byte("csrfsecret")
 	// poolFeesK is the key used to track pool fee payouts.
 	poolFeesK = "fees"
+	// backup is the database backup file name.
+	backupFile = "backup.kv"
```

Comment:
Using a single hard-coded backup name creates a situation where an important backup could be overwritten and lost. Why are we no longer generating unique filenames dynamically?

This is especially risky if we are overwriting the backup file every time dcrpool is stopped

Reply:
There is still the option to backup the db via the admin page, this backup file is what the pool generates automatically. Also it's unlikely the backup gets corrupted during an overwrite because bolt db updates are transactional and are rolled back if there is an error.